[2020-09-04T11:53:18.140] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/find?pagesize=100&page=1&value=zxJSMwgmg4&gameid=&key=CDKID
request client ip:  ::ffff:127.0.0.1
request query:  {"pagesize":"100","page":"1","value":"zxJSMwgmg4","gameid":"1","key":"CDKID"}
response time: 1599191598140
err name: SequelizeDatabaseError
err message: 字段 "zxjsmwgmg4" 不存在
err stack: SequelizeDatabaseError: 字段 "zxjsmwgmg4" 不存在
    at Query.formatError (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee12$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:262:34)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at _byCdkId3 (H:\fancyguo-gm\server\src\app\service\cdk.js:909:35)
    at byCdkId (H:\fancyguo-gm\server\src\app\service\cdk.js:913:35)
    at _callee14$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:239:25)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.findByKey (H:\fancyguo-gm\server\src\app\service\cdk.js:936:27)
    at CDKService._callee11$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:228:21)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.find (H:\fancyguo-gm\server\src\app\service\cdk.js:782:22)
    at _callee7$ (H:\fancyguo-gm\server\src\app\controllers\/cdk.js:62:36)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-04T16:33:44.921] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/find
request client ip:  ::ffff:127.0.0.1
request query:  {"gameid":"1"}
response time: 1599208424920
err name: SequelizeDatabaseError
err message: 字段 "nan" 不存在
err stack: SequelizeDatabaseError: 字段 "nan" 不存在
    at Query.formatError (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee15$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:343:33)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.findByFilter (H:\fancyguo-gm\server\src\app\service\cdk.js:1010:30)
    at CDKService._callee11$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:230:21)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.find (H:\fancyguo-gm\server\src\app\service\cdk.js:782:22)
    at _callee7$ (H:\fancyguo-gm\server\src\app\controllers\/cdk.js:62:36)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at find (H:\fancyguo-gm\server\src\app\controllers\cdk.js:274:22)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
*************** error log end ***************

[2020-09-04T16:52:12.133] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/find
request client ip:  ::ffff:127.0.0.1
request query:  {"gameid":"1"}
response time: 1599209532132
err name: SequelizeDatabaseError
err message: 字段 "nan" 不存在
err stack: SequelizeDatabaseError: 字段 "nan" 不存在
    at Query.formatError (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee15$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:343:33)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.findByFilter (H:\fancyguo-gm\server\src\app\service\cdk.js:1010:30)
    at CDKService._callee11$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:230:21)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.find (H:\fancyguo-gm\server\src\app\service\cdk.js:782:22)
    at _callee7$ (H:\fancyguo-gm\server\src\app\controllers\/cdk.js:62:36)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at find (H:\fancyguo-gm\server\src\app\controllers\cdk.js:274:22)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
*************** error log end ***************

[2020-09-04T16:52:23.000] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/find
request client ip:  ::ffff:127.0.0.1
request query:  {"gameid":"1"}
response time: 1599209543000
err name: SequelizeDatabaseError
err message: 字段 "nan" 不存在
err stack: SequelizeDatabaseError: 字段 "nan" 不存在
    at Query.formatError (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee15$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:343:33)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.findByFilter (H:\fancyguo-gm\server\src\app\service\cdk.js:1010:30)
    at CDKService._callee11$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:230:21)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.find (H:\fancyguo-gm\server\src\app\service\cdk.js:782:22)
    at _callee7$ (H:\fancyguo-gm\server\src\app\controllers\/cdk.js:62:36)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at find (H:\fancyguo-gm\server\src\app\controllers\cdk.js:274:22)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
*************** error log end ***************

[2020-09-04T16:58:31.788] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/find
request client ip:  ::ffff:127.0.0.1
request query:  {"gameid":"1"}
response time: 1599209911788
err name: SequelizeDatabaseError
err message: 字段 "nan" 不存在
err stack: SequelizeDatabaseError: 字段 "nan" 不存在
    at Query.formatError (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee15$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:343:33)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.findByFilter (H:\fancyguo-gm\server\src\app\service\cdk.js:1010:30)
    at CDKService._callee11$ (H:\fancyguo-gm\server\src\app\service\/cdk.js:230:21)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.find (H:\fancyguo-gm\server\src\app\service\cdk.js:782:22)
    at _callee7$ (H:\fancyguo-gm\server\src\app\controllers\/cdk.js:62:36)
    at tryCatch (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at find (H:\fancyguo-gm\server\src\app\controllers\cdk.js:274:22)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (H:\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
*************** error log end ***************

